Games/core/Game.js

```diff
@@ -1203,9 +1203,9 @@ module.exports = class Game {
   setStateShouldSkip(name, shouldSkip) {
     for (let i in this.states) {
       if (this.states[i].name == name) {
-        if (this.states[i].shouldSkip == null) this.states[i].shouldSkip = [];
-
-        this.states[i].shouldSkip.push(shouldSkip);
+        if (this.states[i].skipChecks == null)
+        this.states[i].skipChecks = [];
```

Comment:
if you're making this change, please port all game modes like jotto ghost secret hitler resistance